esy.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "bs-platform",
+  "version": "5.2.0-dev.2",
```

Comment:
Do we need maintain this, if so, which other fields do we need watch?

Reply:
Except for "dependencies", "devDependencies", "resolutions" and "esy" it's a copy of the normal `package.json`, I think we could just remove most fields if we don't want them here.